radio/src/pulses/dsm2_arm.cpp

```diff
@@ -140,7 +137,7 @@ void setupPulsesDSM2(unsigned int port)
       break;
   }
 
-#if !defined(PPM_PIN_HW_SERIAL)
+#if !defined(PPM_PIN_SERIAL)
   if (moduleFlag[port] == MODULE_BIND)
```

Comment:
Althought not really part of this commit, this ifdef looks strange. Binding not supported on Horus?